### DISCUSSION TITLE
muiltiple dirs ignore by `TS_NODE_IGNORE`

### DISCUSSION THREAD

**Discussion by pincman** [TypeStrong/ts-node#1805] [Q&A] [answered]
how to set `TS_NODE_IGNORE` for multi dirs? examples: ['/node_modules','/back']

**Comment by cspotcode** [comment, marked as answer]
It's split on commas and then each is interpreted as a regex.  You can also see how the default regex is written as an example.  Here are references to the source code to show how it is interpreted.

https://github.com/TypeStrong/ts-node/blob/bf13086752f818421ddd30e430b6e2fed706764b/src/index.ts#L447
https://github.com/TypeStrong/ts-node/blob/bf13086752f818421ddd30e430b6e2fed706764b/src/util.ts#L47-L51
https://github.com/TypeStrong/ts-node/blob/bf13086752f818421ddd30e430b6e2fed706764b/src/index.ts#L719-L726
